phlop/testing/test_cases.py

```
@@ -122,11 +123,15 @@ def test_cmd_fn(c, t):
 
 def load_py_test_cases_from_cmake(ctest_test):
     ppath = ctest_test.env.get("PYTHONPATH", "")
-    bits = ctest_test.cmd.split(" ")
+    bits = shlex.split(ctest_test.cmd)
     idx = [i for i, x in enumerate(bits) if "python3" in x][0]
```

Ruff (0.15.21)

[warning] Prefer `next(i for i, x in enumerate(bits) if "python3" in x)` over single element slice

Replace with `next(i for i, x in enumerate(bits) if "python3" in x)`

(RUF015)